builds/install/arch-specific/linux/Makefile.in

```diff
@@ -76,20 +76,17 @@ PackageDir:=$(shell if [ -d $(RedhatPackageDir) ]; then echo $(RedhatPackageDir)
 
 PkgSrcDir=$(InstallRoot)/$(ArchType)
 ScriptDir=$(GEN_ROOT)/install/scripts/
-RPMSrcDir=$(GEN_ROOT)/install/scripts
 
 
 TarInstallDir=$(GEN_ROOT)/install
 
-#Target for required libraries detection
-RpmRequiredVersionTarget=$(GEN_ROOT)/firebird/bin/isql
 #Curses library
 TermLib=@TERMLIB@.so
 
 
 dist : packages
 
-packages: tarfile rpmfile debugfile
+packages: tarfile debugfile
```

Comment:
Why are you removing things deprecated later from 2.5 branch?

Reply:
I did a diff on this file with 3.0 branch and merged the removal of rpmfile building (related to http://tracker.firebirdsql.org/browse/CORE-5180) , i have asked also on devel list if it is ok